about.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+       <title>All about the PSS conference</title>
+</head>
+<body>
+    <section>
+        <div class="link-bar">
+            <ul>
+                <li><a href=""><img src="./images/Vector4.png" class="alt-img"></a></li>
+                <li><a href=""><img src="./images/Vector2.png" class="alt-img"></a></li>
+                <li>Ikoote</li>
+                <li>MyPage</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+    </section>
+    <section class="menu-a">
+        <div class="nav-menu-a">
+            <div><a href="./index.html"><img src="./images/img-logo.jpg">ikoote PSS</a></div>
```

Comment:
[OPTIONAL] Well done on the `navbar` 👏🏽 it really helps the user navigate through your project but it could look even better 🤔 this doesn't look quite good yet as the text is not styled and is not properly placed. See below 👇🏽 
![Screenshot 2023-02-09 at 07 28 05](https://user-images.githubusercontent.com/45739963/217812814-11399ce4-5d8c-4375-b7ef-7e6179c716f9.png)

Therefore I'd suggest either removing it completely or just adjusting its style. It's very optional but would make the project better. 🎨